Add Google Analytics

dh4u opened the pull request:
https://app.asana.com/0/1192620845659667/1199595507254378/f

I did have one question about the packages that suggested a npm audit fix that I put in the Asana card.

<img width="574" alt="image" src="https://user-images.githubusercontent.com/40501523/104961084-a18f5a80-59a3-11eb-9fa7-854dca76e6ca.png">

dh4u committed:
Add Google Analytics

https://app.asana.com/0/1192620845659667/1199595507254378/f

atlisg commented on components/utils/analytics.js:
```diff
@@ -0,0 +1,10 @@
+import ReactGA from 'react-ga';
+
+export const initGA = () => {
+  ReactGA.initialize('UA-113130296-3');
```
This is the FOOD-E ID, the correct ID is `UA-113130296-2`

dh4u replied:
Updated.... https://github.com/ethicodeorg/website/commit/dcf8cea221b6212ac44c0bbf02380925d44374d2

atlisg reviewed (changes requested):
Just the ID and we're good to go.
Note: Because this is a slightly different setup for the Layout component, I'm not sure if the pages will be tracked individually but let's just see what happens. If this only logs the main page, we can fix that quickly

Let's do the npm audit in a separate PR. Feel free to go for it if you like. :)

dh4u committed:
Update analytics.js

Just the ID and we're good to go.
Note: Because this is a slightly different setup for the Layout component, I'm not sure if the pages will be tracked individually but let's just see what happens. If this only logs the main page, we can fix that quickly

Let's do the npm audit in a separate PR. Feel free to go for it if you like. :)
components/utils/analytics.js
import ReactGA from 'react-ga';

 export const initGA = () => {
  ReactGA.initialize('UA-113130296-3');

 @atlisg atlisg 3 days ago  Member
This is the FOOD-E ID, the correct ID is UA-113130296-2

atlisg reviewed (approved):
Looks good, feel free to merge. 👍
